vetiver/pin_read_write.py

```diff
@@ -51,14 +51,20 @@ def vetiver_pin_write(board, model: VetiverModel, versioned: bool = True):
         "with vetiver.model_card()",
     )
 
+    # convert older model's ptype to prototype
+    if hasattr(model, "ptype"):
+        model.prototype = model.ptype
+
     board.pin_write(
         model.model,
         name=model.model_name,
         type="joblib",
         description=model.description,
         metadata={
             "required_pkgs": model.metadata.get("required_pkgs"),
-            "ptype": None if model.ptype is None else model.ptype().json(),
+            "ptype": None
```

Comment:
Should this change so that the metadata that is pinned has an element called "prototype" now?

Reply:
Yep! Updated!